### ISSUE TITLE
[wg/pat] Privacy-preserving advertising

### ISSUE THREAD

**Issue by wseltzer** [w3c/strategy#222]
[WG Charter 2022 horizontal review [below](https://github.com/w3c/strategy/issues/222#issuecomment-1134669362).
[2022 AC review](https://github.com/w3c/strategy/issues/222#issuecomment-1480335217).
And [2023 AC review](https://github.com/w3c/strategy/issues/222#issuecomment-1764969345)]

The [Improving Web Advertising Business Group](https://github.com/w3c/web-advertising) is exploring proposals to support advertising and monetization on the open web without individually-identified cross-site or web-wide tracking.

* A [table of business use cases matched with technical proposals](https://github.com/w3c/web-advertising/blob/master/support_for_advertising_use_cases.md)

* Relevant proposals are being discussed and incubated in WICG and PrivacyCG.

* When ready, work may migrate to WebAppSec, HTML, or a new group.

**Comment by chaals** [comment]
Is this something obviously different from #62? @draggett?

**Comment by dontcallmedom** [comment]
https://www.w3.org/community/patcg/ was launched in October to identify proposals towards standardization and a supporting WG charter

**Comment by samuelweiler** [comment]
Draft WG charter: https://github.com/patcg/patwg-charter

I'm moving this pipeline card from "Investigation" to "Evaluation"

**Comment by dontcallmedom** [comment]
direct link to charter https://patcg.github.io/patwg-charter/charter.html

**Comment by samuelweiler** [comment]
The draft charter is ready for horizontal review.

# Charter Review

[Private Advertising Technology Working Group Charter](https://patcg.github.io/patwg-charter/charter.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

 - [x] New WG 

Advance notice sent: https://lists.w3.org/Archives/Public/public-new-work/2022May/0009.html and https://lists.w3.org/Archives/Member/w3c-ac-members/2022AprJun/0037.html

Communities suggested for outreach: Improving Web Advertising BG, Privacy CG

Known or potential areas of concern: 

Where would charter proponents like to see issues raised?   https://github.com/patcg/patwg-charter/issues

Anything else we should think about as we review?  This work has been - and future work items will be - incubated in the Privacy CG and/or Private Advertising Technology (PAT) CG.  The charter was developed by the community in the PAT CG.

**Comment by himorin** [comment]
- line of `Brief background of ` at right above '1. Scope' is remaining as template but not removed??
- It seems a link to 'Private Attribution Measurement' is not set? (I could find only [repository for discussion](https://github.com/patcg/private-measurement) but not spec development one, although..)

**Comment by himorin** [comment]
no comment/request from i18n

**Comment by samuelweiler** [comment]
Sent for AC review in August 2022: https://lists.w3.org/Archives/Member/w3c-ac-members/2022JulSep/0033.html

Returned to CG for further work October 2022: https://lists.w3.org/Archives/Member/w3c-ac-members/2022OctDec/0008.html

**Comment by samuelweiler** [comment]
The PAT CG has revised the charter and is [ready](https://github.com/patcg/meetings/tree/main/2023/05/02-telecon) for it to be considered again.

**Comment by michael-n-cooper** [comment]
Reviewing from TiLT role, some comments:

1. The link to the one normative spec is dead; AC reps must be able to see the proposed doc. I haven't checked all links, this should be done.
2. I'm not comfortable with "The Working Group intends to add other normative specifications as the technology becomes ready". The charter should specify what it will work on, and recharter if adding new work. An editorial change that would address my concern and might meet the intent anyways would be to clarify that the deliverables under incubation would be added in a future charter period of this group.
3. Section 2.3 Timeline hasn't been completed. It needs to say something, and can have a reference to an external resource for better info.
4. This may need updates to the latest charter template, in the [template diff](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https%3A%2F%2Fpatcg.github.io%2Fpatwg-charter%2Fcharter.html%23) the Success Criteria section in particular seems out of date.
5. I'm curious about section 10 Anti-trust. This is not something I've seen in a charter before, and I don't know the reasons or implications. It would be helpful to know more about why that's there and what review was involved.

**Comment by mnot** [comment]
I tend to agree re: section 10; charters shouldn't be setting policies like this.

**Comment by plehegar** [comment]
section  10 should be removed since it's applicable no matter what.

**Comment by svgeesus** [comment]
Strong -1 from me, this charter is only partly finished and is not at all ready for AC review.

1. I agree with all of the points that @michael-n-cooper makes above, some of which I also made on the most recent TiLT call and which have not been addressed.

2. The sole deliverable has an empty link:

```html
<dt id="private-attribution" class="spec"><a href="#)">Private Attribution Measurement</a></dt>
```
and is missing all of the [required information for a rec-track deliverable](https://w3c.github.io/charter-drafts/charter-template.html#deliverables) such as the **Draft state**,  **Adopted Draft**, **Exclusion Draft**, and **Exclusion Draft charter**.

3. The Motivation and Background section is missing

4. It is unclear why a novel term "Design Team" is introduced, whose policies "for example" are largely undefined. If this is a task force (subset of the WG, with identical policies) say so.

5. the `[team contact name]` is missing

In general, please start again with a fresh, current copy of the charter template; add in the information from the current charter draft, and re-open this issue when the charter is actually ready for review.
5. 
7.

**Comment by plehegar** [comment]
Timing out on getting from PING for this charter.

**Comment by plehegar** [comment]
@svgeesus closed it by mistake. We're waiting on @samuelweiler to address the comments he received. If you do have last minute comments, please feel free to bring them up. Keep in mind that the next step after that will be to start the AC Review of the charter.

**Comment by chrisn** [comment]
I am concerned by the change in the charter around documenting all known privacy considerations, and moreso by the change in the charter template, see https://github.com/w3c/charter-drafts/pull/415 and https://github.com/patcg/patwg-charter/pull/60, so I'm raising this here, as input to PING consideration, and also in the interest of resolving ahead of AC review.

**Comment by samuelweiler** [comment]
@michael-n-cooper 

> 1. The link to the one normative spec is dead; AC reps must be able to see the proposed doc. I haven't checked all links, this should be done.

The CG has has not yet adopted a baseline document for the attribution measurement deliverable.  I have added the document state clarification ("No Draft") *and* provided a pointer to the repository for discussion.

> 2. I'm not comfortable with "The Working Group intends to add other normative specifications as the technology becomes ready". The charter should specify what it will work on, and recharter if adding new work. An editorial change that would address my concern and might meet the intent anyways would be to clarify that the deliverables under incubation would be added in a future charter period of this group.

The harder we make it to bring in-scope items from the CG into the WG, the more likely the work will simply happen in a CG.

The PAT CG crafted a very specific, narrow, and highly-refined Scope statement that they expect to describe the work the WG will adopt from incubation (from the CG).  Having spent that energy on the Scope, they (and I) don't want a do full recharter merely to add a single item within that scope.  We would of course document the added items at the next rechartering cycle.

@siusin or I may say more about this shortly.

> 3. Section 2.3 Timeline hasn't been completed. It needs to say something, and can have a reference to an external resource for better info.

Fleshed out to the extent possible given only one named deliverable.

> 4. This may need updates to the latest charter template, in the [template diff](https://services.w3.org/htmldiff?doc1=https://w3c.github.io/charter-drafts/charter-template.html&doc2=https%3A%2F%2Fpatcg.github.io%2Fpatwg-charter%2Fcharter.html%23) the Success Criteria section in particular seems out of date.

The CG did some pretty serious [customization of this section.](https://github.com/patcg/patwg-charter/pull/59)  If there are particular template updates you think they overlooked, call those out, but in general the departure from the template is deliberate.

> 5. I'm curious about section 10 Anti-trust. This is not something I've seen in a charter before, and I don't know the reasons or implications. It would be helpful to know more about why that's there and what review was involved.

IIRC, this was added in response to a request from James Rosewell at https://github.com/patcg/patwg-charter/issues/10. W3M has decided that this will be removed.

**Comment by samuelweiler** [comment]
@svgeesus 

> Strong -1 from me, this charter is only partly finished and is not at all ready for AC review.
> 
> 1. I agree with all of the points that @michael-n-cooper makes above, some of which I also made on the most recent TiLT call and which have not been addressed.

See above.

> 2. The sole deliverable has an empty link:
> 
> ```
> <dt id="private-attribution" class="spec"><a href="#)">Private Attribution Measurement</a></dt>
> ```
> 
> and is missing all of the [required information for a rec-track deliverable](https://w3c.github.io/charter-drafts/charter-template.html#deliverables) such as the **Draft state**, **Adopted Draft**, **Exclusion Draft**, and **Exclusion Draft charter**.

As above, the CG has has not yet adopted a baseline document for the attribution measurement deliverable. I have added the document state clarification ("No Draft") and provided a pointer to the repository for discussion.

> 3. The Motivation and Background section is missing

I'm not hearing that the charter is impenetrable and *more* text is needed, so my first thought was to copy and minimally edit text from the "mission" line, even though it would be duplicative.

Intriguingly, this heading was added to the template in response to @himorin's [suggestion](https://github.com/w3c/charter-drafts/issues/383#issue-1085928043) that the prompt for duplicative text be removed.  Instead we now have another section for it.

Acknowledging that this header was introduced to the template in February 2023, long after work on this charter started, I suggest moving forward without adding this section.

> 4. It is unclear why a novel term "Design Team" is introduced, whose policies "for example" are largely undefined. If this is a task force (subset of the WG, with identical policies) say so.

The CG probably chose that term because of other-SDO experiences.  The chairs have no objection to adopting the W3C terminology, and we have updated the charter accordingly.

> 5. the `[team contact name]` is missing

Fixed.  Also updated the copyright.

> In general, please start again with a fresh, current copy of the charter template; add in the information from the current charter draft, and re-open this issue when the charter is actually ready for review.

While that's a good idea for something multiple years out of date (e.g. rechartering), given that this is "coming back from the AC" rework with 1) a fairly modern underlying template and 2) deliberate changes (e.g. in the success criteria, as documented above) that required some careful argument, that's probably not the best approach this time around.

**Comment by siusin** [comment]
The chairs have been working on the FOs in GitHub and this [team-only Google Doc](https://docs.google.com/document/d/18yUYteiF97XZYBP4Zc1XrardioWER00vBJ-1jvfAHkY/edit#heading=h.6so0rigyfm6e) (it's not public because it contains member-only info). Hope this will be helpful to your next Strategy meeting discussion.

**Comment by pes10k** [comment]
I think its important for [this](https://github.com/patcg/patwg-charter/issues/61) issue to be addressed. Right now the group has a negative scope (to pursue advertising solutions that are appropriate, where appropriate doesn't mean X or Y).  This seems odd and overly broad in defining a scope for a group. I think the charter should state what privacy goals or boundaries it group is aiming to work within, not an incomplete, unbounded list of things it won't do.

**Comment by plehegar** [comment]
@AramZS @seanturner I believe that we're currently stuck for the PAT WG charter. The CG is rejecting the Team request to update the proposal charter draft further.

The W3C Strategy team believes that the charter, as currently written, is an open ended charter that is not ready to commit to an actual technical proposal yet. At best, it links to an empty repository that hasn't been updated for more than a year.

Either we propose to set up a Working Group:
1. to work on one or more actual technical proposals (and may adopt further proposals later),
2. or, to explore the different proposals and recharter the Working Group with the actual technical proposals once we know how to move forward (similar to what was done with the Web Fonts charter in the past).

Should we get together to figure this out?

**Comment by mnot** [comment]
@plehegar could you link to where these requirements for new work are documented? Thanks.

**Comment by seanturner** [comment]
@plehegar
I checked with @AramZS can we meet next week Tuesday (July 8) after 2pm eastern or Wednesday (July 9) after 2pm.

**Comment by plehegar** [comment]
@mnot , we don't have such list. We have a charter template, which gets adjusted based on feedback we receive during AC reviews. In our experience, open-ended charter for a complete new group will raise a lot of eyebrows from the Membership. There was a recent discussion around the privacy working group which had similar issues for example.

**Comment by plehegar** [comment]
this will need to loop through the CG but I believe we may have a path forward here:
* refine the description of the PAM deliverable to narrow it down a bit more
* resync the success criteria with the charter template
* refine the coordination with IETF/ECMA/WHATWG for the purposes of demonstrating wide review before moving to Candidate Recommendation

**Comment by siusin** [comment]
One sentence was dropped from the success criteria of the charter template:
[[
This Working Group expects to follow the TAG [Web Platform Design Principles](https://www.w3.org/TR/design-principles/)
]]
Is there any story behind it or shall we try to be in sync with the template?

**Comment by svgeesus** [comment]
I have the same question as @siusin 

That one point aside, the charter looks much better now.

**Comment by plehegar** [comment]
Pull request to re-align the proposed charter with the template is at https://github.com/patcg/patwg-charter/pull/74

**Comment by samuelweiler** [comment]
[Announcement of AC review](https://lists.w3.org/Archives/Member/w3c-ac-members/2023OctDec/0016.html)

[Snapshot for review](https://www.w3.org/2023/10/PROPOSED-PATWG-charter.html)
[WBS poll](https://www.w3.org/2002/09/wbs/33280/PAT-WG-Charter/)

**Comment by plehegar** [comment]
update: we have one Formal Objection and the Team is investigating.

**Comment by martinthomson** [comment]
@plehegar just confirming, but are these the same objection with different formatting, or am I missing something?

**Comment by plehegar** [comment]
Council was convened on March 3.

**Comment by plehegar** [comment]
The Team announced convening a Council on [March 3rd](https://lists.w3.org/Archives/Member/w3c-ac-members/2024JanMar/0041.html).

**Comment by plehegar** [comment]
The Council was [announced](https://lists.w3.org/Archives/Member/w3c-ac-members/2024JulSep/0063.html). [Team report](https://www.w3.org/2024/07/team-report-pat-wg-fo.html) is available.

**Comment by plehegar** [comment]
The CEO overruled the formal objection and we're now proceeding with launching the Working Group.

**Comment by plehegar** [comment]
Group was [announced](https://lists.w3.org/Archives/Public/public-new-work/2024Nov/0005.html).
